--- a/target/linux/qualcommax/dts/ipq6000-m2.dts
+++ b/target/linux/qualcommax/dts/ipq6000-m2.dts
@@ -51,7 +51,12 @@
 };

 &wifi {
 	status = "okay";
 	qcom,ath11k-fw-memory-mode = <1>;
 	qcom,ath11k-calibration-variant = "ZN-M2";
 };
+
+&usb2 { status = "disabled"; };
+&usb3 { status = "disabled"; };
+&qusb_phy_0 { status = "disabled"; };
+&qusb_phy_1 { status = "disabled"; };
+&ssphy_0 { status = "disabled"; };
